<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>253</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>236</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>236</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TelemFFB MultiLauncher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="logowidget" native="true">
      <property name="minimumSize">
       <size>
        <width>218</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>218</width>
        <height>82</height>
       </size>
      </property>
      <widget class="QLabel" name="image_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>218</width>
         <height>82</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>218</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="pixmap">
        <pixmap>image/vpforcelogo.png</pixmap>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>     Joystick</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tb_joystick">
        <property name="text">
         <string>2055</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_pedals">
        <property name="text">
         <string>Pedals</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tb_pedals"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cb_collective">
        <property name="text">
         <string>Collective</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tb_collective"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter the USB address (default: 2055)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_autolaunch">
      <property name="text">
       <string>Launch when opened</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_launch">
      <property name="text">
       <string>Launch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
